src/components/Card/Card.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import {useNavigate} from 'react-router-dom';
+import {ListItemColumn, Teams, UserData} from 'types/types';
+import {CardContainer} from './Card.styled';
+
+interface CardProps {
+    id: string;
+    url?: string;
```

Comment:
`id` should be required, because if not, can't be tested (it is used in the `data-testid` property